addons/remove-sprite-confirm/addon.json

```diff
@@ -1,15 +1,47 @@
 {
-  "name": "Sprite deletion confirmation",
-  "description": "Asks if you're sure when deleting a sprite inside a project.",
+  "name": "Editor confirmations",
+  "description": "Asks if you're sure when deleting a sprite/costume/sound inside a project.",
   "userscripts": [
     {
       "url": "userscript.js",
       "matches": ["projects"]
     }
   ],
+  "settings": [
+    {
+      "name": "Sprites",
+      "id": "sprites",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Costumes and backdrops",
+      "id": "costumes",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Sounds",
+      "id": "sounds",
+      "type": "boolean",
+      "default": true
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "fix-z-index.css",
+      "matches": ["projects"]
+    }
+  ],
   "dynamicEnable": true,
   "dynamicDisable": true,
   "versionAdded": "1.0.0",
   "tags": ["editor", "recommended"],
-  "enabledByDefault": false
+  "enabledByDefault": false,
+  "latestUpdate": {
+    "version": "1.28.0",
+    "isMajor": true,
+    "temporaryNotice": "You can now select, to ask for confirmations when deleting costumes and sounds.",
+    "newSettings": ["sprites", "costumes", "sounds"]
```

Comment:
Do we want `sprites` to be a new setting? Also, maybe `costumes` and `sounds` shouldn't be true by default, it'd enable it for everyone who already enabled the addon.

Reply:
IMAO `sprites` should be new setting, but I agree that costumes and sounds should be false by default.

Reply:
WL said: `not for now`, so maybe we can do that in another PR.